src/main/java/racingcar/RacingCarGame.java

```diff
@@ -20,15 +22,30 @@ public void start() {
         Input input = inputHandler.getInput();
         List<RacingCar> racingCars = input.toRacingCars();
 
+        if (racingCars.isEmpty()) {
+            return;
+        }
+
         for (int i = 0; i < input.numberOfCountToTry; i++) {
             tryToMoveRacingCars(racingCars);
-            resultView.print(racingCars);
+            resultView.printProgress(racingCars);
         }
+
+        List<RacingCar> winners = getWinners(racingCars);
+        resultView.printWinners(winners);
     }
 
     private void tryToMoveRacingCars(List<RacingCar> racingCars) {
         for (RacingCar racingCar : racingCars) {
             racingCar.tryToMove();
         }
     }
+
+    private List<RacingCar> getWinners(List<RacingCar> racingCars) {
+        Collections.sort(racingCars);
+        RacingCar winner = racingCars.get(0);
+        return racingCars.stream()
+                .filter(rc -> rc.isSameMoveCount(winner))
+                .collect(Collectors.toList());
```

Comment:
다음과 같이 Collections.unmodifiablelist()를 활용하여 read-only로 리턴하여 외부에서 값의 변조를 막도록 하면 좋을 것 같아요 :)
```suggestion
                .collect(Collectors.collectingAndThen(Collectors.toList(), Collections::unmodifiableList));
```